src/lib/Preview.js

```diff
@@ -588,6 +596,32 @@ class Preview extends EventEmitter {
             });
     }
 
+    /**
+     * Setup additional configuration for the logger.
+     *
+     * @param {Object} config - Configures log level and network layer.
+     * @param {CONSOLE_LEVELS|string} [config.consoleLevel] - Level to set for writing to the browser console.
+     * @param {boolean} [config.savingEnabled] - If true, allows saving of logs to a network layer.
+     * @param {string} [config.logURL] - Full url to save logs to. Can instead use appHost with logEndpoint (see below)
+     * @param {string} [config.appHost] - Base URL to save logs to. Is combined with logEndpoint (below)
```

Comment:
Are we still going for the idea of allowing people to save to their own back end? That might be better for a v2/after seeing some demand

Reply:
Agreed that this will be better suited to V2. Leaving docs and still using it bc I want to use our Preview options to configure this, as it's easier when switching between dev env and live